pkg/controllers/awsloadbalancercontroller/controller.go

```diff
@@ -73,7 +73,8 @@ type AWSLoadBalancerControllerReconciler struct {
 //+kubebuilder:rbac:groups="apps",resources=deployments,namespace=system,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups="",resources=serviceaccounts,namespace=system,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=rbac.authorization.k8s.io,namespace=system,resources=roles;rolebindings,verbs=get;list;watch;create;update;patch;delete
-//+kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=clusterroles;clusterrolebindings,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=clusterrolebindings,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=clusterroles,verbs=bind,resourceNames=aws-load-balancer-operator-controller-role
```

Comment:
Doesn't the permissions here need to be updated? The controller will need permissions to get the clusterrole as well right?

Reply:
Yes they do, forgot to do it. Hm, I'm wondering how e2e didn't catch that.